compilator/compilator.go

```diff
@@ -703,8 +703,14 @@ func (c *Compilator) removeCompiledPackages(packages model.Packages) (model.Pack
 
 		if compiled {
 			close(c.signalDependencies[pkg.Fingerprint])
+			if verbose {
+				c.ui.Printf("found %s in %s\n", color.YellowString(pkg.Name), pkg.GetPackageCompiledDir(c.hostWorkDir))
```

Comment:
It feels like `isPackageCompiledHarness` might want to return `GetPackageCompiledDir`, so that it can change and still have the correct report… thoughts?